editor/sidebar/post-visibility/index.js

```diff
@@ -71,6 +74,21 @@ class PostVisibility extends Component {
 		this.setState( { opened: false } );
 	}
 
+	handleBlur( event ) {
+		if ( this.state.opened && event.relatedTarget !== null && ! this.dialog.contains( event.relatedTarget ) ) {
```

Comment:
This needs to be changed anyways, since `relatedTarget ` doesn't work on Safari 10 when using VoiceOver (see comment on the issue). We need to find an alternative solution.